scripts/make_orog.py

```
@@ -0,0 +1,153 @@
+
+    # Run orog_gsl only when using GSL's orography drag suite
+    ccpp_phys_suite = expt_config["workflow"]["CCPP_PHYS_SUITE"]
+    orog_drag_suites = [
+        "FV3_RAP",
+        "FV3_HRRR",
+        "FV3_GFS_v15_thompson_mynn_lam3km",
+        "FV3_GFS_v17_p8",
+    ]
+    if ccpp_phys_suite in orog_drag_suites:
+        orog_gsl_driver = run_driver(
+            driver_class=OrogGSL, config_file=config_file, key_path=key_path
+        )
+        output_files = [
+            orog_gsl_driver.rundir / f"{cres}_oro_data_{i}.tile7.halo0.nc"
+            for i in ("ss", "ls")
+        ]
+        link_files(dest_dir=fix_lam_path, files=output_files)
+
+    # Run filter_topo
+    run_driver(driver_class=FilterTopo, config_file=config_file, key_path=key_path)
+
+    # Run shave for 0- and 4-cell-wide halo
+    for sub_path in ["shave0", "shave4"]:
+        driver = run_driver(
+            driver_class=Shave,
+            config_file=config_file,
+            key_path=[*key_path, sub_path],
+        )
+
+    # Link shave output to fix directory (shave config points to task dir, not shave rundir)
+    link_files(
+        dest_dir=fix_lam_path,
+        files=glob.glob(str(task_rundir / f"{cres}*.nc")),
+    )
+
+    # Mark the successful completion of the script on disk.
+    Path(task_rundir / "make_orog_task_complete.txt").touch()
+
+
+def run_driver(driver_class, config_file, key_path):
+    """
+    Initialize and run the provided UW driver.
+
+    Return the configured object.
+    """
+    driver = driver_class(
+        config=config_file,
+        key_path=key_path,
+    )
+    rundir = Path(driver.config["rundir"])
+    logging.info(f"Will run {driver.driver_name()} in {rundir}")
+    driver.run()
+
+    if not (rundir / f"runscript.{driver.driver_name()}.done").is_file():
+        logging.error(
+            f"Error occurred running {driver.driver_name()}. Please see component error logs."
```

`pylint` may ultimately complain about this, as I think it recommends the
```
logging.error("Error occurred running %s. Please see component error logs.", driver.driver_name())
```
style, with delayed string interpolation. That could be dealt with later if/when it happens.

---

We've ignored this error in SRW. 😬